### ISSUE TITLE
Problems with the integration

### ISSUE THREAD

**Issue by ramosjean1** [AdrianJPT/Netbox-Zabbix-Integration#1]
I'm implementing the integration but I can't make it work for me. I have Zabbix and Netbox installed under Debian 12 in different virtual machines and therefore with different IPs. I have done all the steps but when creating the devices in Netbox they are not reflected in Zabbix. What URL is used in the webhooks is it a random ipv4 or must a specific one be configured? The other thing that I have done is changes in the credentials.py file, changing the URLs to those of the APIs and nothing that works. Please I need help.

**Comment by jdinocco** [comment]
Hi! do you configure the internal webhooks to the docker endpoint? 

![image](https://github.com/AdrianJPT/Netbox-Zabbix-Integration/assets/6908410/63a6dd49-bbbe-4a3b-b376-4db772c2c6c8)

https://demo.netbox.dev/static/docs/additional-features/webhooks/

That is the action that trigger the automation.

**Comment by ramosjean1** [comment]
Yes, I already configured the Webhooks as explained in the video. But, it doesn't work for me, is that url you used taken randomly or should it be set in some repository file?

**Comment by jdinocco** [comment]
That's weird, you can share how you have configured the webhooks and the credentials.py file?

Thanks!

**Comment by ramosjean1** [comment]
https://i.postimg.cc/h4whm8xv/credentials-py.png

**Comment by ramosjean1** [comment]
https://i.postimg.cc/m2jLwQ17/example.png

**Comment by ramosjean1** [comment]
Attach screenshots of what you requested. In the Webhooks I am testing with different URLs. I appreciate all the help you are giving me.

**Comment by thuvh** [comment]
:( your url should be http://192.168.1.12:5000/action, if you deploy api in same host 

> https://i.postimg.cc/m2jLwQ17/example.png
